src/neo/SmartContract/ContractParametersContext.cs

```diff
@@ -232,6 +232,17 @@ public Witness[] GetWitnesses()
             return witnesses;
         }
 
+        public IEnumerable<UInt160> GetPendingWitnesses()
+        {
+            for (int i = 0; i < ScriptHashes.Count; i++)
+            {
+                if (ContextItems.ContainsKey(ScriptHashes[i]))
```

Comment:
I think it's useless. Because a multi-signature account has only one script hash.

Reply:
But a transaction could be signed by multiple accounts

Reply:
You need to check whether the multi-signature is completed. I think that checking script hashes doesn't work for you.

Reply:
You are right, take a look, I think that I can do it with `GetSignatures`